api/database/seeders/DatabaseSeeder.php

```diff
@@ -36,6 +36,7 @@ public function run(): void
             UserRandomSeeder::class,
             AssessmentResultRandomSeeder::class,
             SearchRequestRandomSeeder::class,
+            JobPosterTemplateSeeder::class,
```

Comment:
Since this seeder produced "real" data I'd move this up to the "standard platform data" section and also add it to the UAT, CI, and prod seeders.

Reply:
We should probably get rid of JobPosterTemplateRandomSeeder, too.  We don't really need it anymore.  If someone wants to seed a bunch of random templates they can call the factory directly.

Reply:
Yep
https://github.com/GCTC-NTGC/gc-digital-talent/pull/11523/commits/6db830b7b64f7121e71f58e47b15ddf8c922e7bf
https://github.com/GCTC-NTGC/gc-digital-talent/pull/11523/commits/2f09d5fc79bf49f3cfec3d5a8715f1f940108fba
Apparently we don't have a `TeamTestSeeder`, I think that one was supposed to be `TeamSeeder`